### PR TITLE
Using sso-api-proxy in example application

### PR DESCRIPTION
**Pull request by minond**
Introduces a new command, `npm run example`, which will first optionally build the application when it hasn't been built already, then start an SSO API proxy using [`@mxenabled/sso-api-proxy`](https://www.npmjs.com/package/@mxenabled/sso-api-proxy). The `mx-sso-api-proxy` command will ask the user to enter their API and user settings if it doesn't find a configuration file or environment variables. Here's what that experience looks like:

```console
$ npm run example
? Client ID 496b5ddc-7608-49fe-8c06-56d8ebfe4b1f
? API key 7f683eea8aac0245c56bb373572e6aea519a8e2c
? Default user guid USR-081ff65e-3087-4cc2-a2c4-365354e1e6cb
? Environment Development
? Would you like to save this configuration? Yes
Saving configuration to ./mx-sso-api-proxy-rc.yaml
Service local files
Running server on port 8089
GET /example/ 304 - - 3.535 ms
GET /example/shared.css 304 - - 2.022 ms
GET /example/shared.js 304 - - 1.186 ms
GET /dist/amd/index.js 304 - - 0.580 ms
POST /user/widget_urls 200 627 - 246.405 ms
```

With this, a user would only have to clone the repo and run `npm install`, then `npm run example` to run example app.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@ cypress/downloads
 cypress/screenshots
 cypress/videos
 dist
+mx-sso-api-proxy-rc.yaml
```
Comment (sambev) [inline .gitignore:9]: Is this a config file or something? Is is something we should document?

Comment (sambev) [reply .gitignore:9]: I answered my own question, seems like that is from the sso api proxy. Should we add a link to that `https://gitlab.mx.com/mx/widget-sso-api-proxy#configuration` in the example readme?

Comment (minond) [reply .gitignore:9]: It's the config for https://github.com/mxenabled/sso-api-proxy. That tool will walk the user through how to configure it, but maybe we link to that project in the example app's readme.

Comment (minond) [reply .gitignore:9]: Yup, thinking the same.

Comment (minond) [reply .gitignore:9]: Done @sambev

**Pull request merged**